cli/flox-rust-sdk/src/models/lockfile.rs

```diff
@@ -347,6 +355,25 @@ struct LockedGroup {
     page: CatalogPage,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+#[cfg_attr(test, derive(proptest_derive::Arbitrary))]
+pub struct Compose {
+    /// The composing environment's manifest that was on disk at lock-time.
+    pub composer: Manifest,
+    /// Metadata and manifests for the included environments in the order
+    /// that they were specified in the composing environment's manifest.
+    pub include: Vec<LockedInclude>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+#[cfg_attr(test, derive(proptest_derive::Arbitrary))]
+pub struct LockedInclude {
+    pub manifest: Manifest,
+    pub descriptor: IncludeDescriptor,
```

Comment:
followup: we're going to make name optional on IncludeDescriptor and add `name` to this struct, right?

Reply:
Just slipped my mind to do as part of this PR. I'll add that in real quick.

Reply:
Done